nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -247,7 +249,22 @@ getMergeableGraph(std::shared_ptr<const GraphRepresentation> graph,
 
 void IniGraphInterpreter::serialize(
   std::shared_ptr<const GraphRepresentation> representation,
-  const std::string &out) {}
+  const std::string &out) {
+
+  std::vector<IniSection> sections;
+  for (const auto &ln : representation->getSorted()) {
```

Comment:
any reason to use `getSorted()`?

Reply:
The ini representation is independent of the order of the sections.

Reply:
AFAIK, there are some bugs if first layer is not input layer or output layer is before input layer and so on..
Even though above is sorted out, it is neater and more human readable to make it sorted.

Reply:
> AFAIK, there are some bugs if first layer is not input layer or output layer is before input layer and so on..

I think this was resolved when the network_graph was refactored (not yet with backbone though).

Reply:
> Even though above is sorted out, it is neater and more human readable to make it sorted.

Then lets use unsorted representation in debug mode to make sure that the all styles work.

Reply:
Let me have a todo instead, I am also testing backbone.

Reply:
ok.